physionet-django/notification/templates/notification/email/mailto_contact_supervisor.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}{% autoescape off %}
+Dear Prof. {{ application.reference_name }},
```

Comment:
Same comment as above...don't think we want to hardcode prof, but fine to leave in for now if it's necessary.

Reply:
He said he always defaults to Prof. since this is a mailto link to generate his own email, he can change this afterwards in his own email client